### PR TITLE
FixFileGet: fix file get test failure

### PR DESCRIPTION
**Pull request by steveyeom**

### CODE REVIEW

File: cmd/btfs/functest.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	testfile        = "QmZHeNJTU4jFzgBAouHSqbT2tyYJxgk6i15e7x5pudBune"
+	testfile        = "QmPdsdr4JznLs2FFFhjDbMzrnXmNfny8wpKXETKfbXdeK7"
```
Comment (jin-tron) [inline cmd/btfs/functest.go]: Why we need to update this value? Is there hash function change?

Comment (wxue) [reply cmd/btfs/functest.go]: > Why we need to update this value? Is there hash function change?
yes, metadata has been added to the file

**Comment by taiyangc** [comment]
@steveyeom Can we track this in a ticket and close this PR for now? You will have changes coming soon that may invalidate this.

**Comment by steveyeom** [comment]
BTFS-911 is created.

**Pull request closed without merging**